scene/gui/popup_menu.cpp

```diff
@@ -1019,6 +1022,16 @@ void PopupMenu::_set_items(const Array& p_items){
 
 }
 
+// Hide on Select determines whether or not the popup will close after item selection
+void PopupMenu::set_hide_on_item_selection(bool p_bool) {
```

Comment:
Prefer using `p_enabled` to `p_bool`, that name is used throughout the code and slightly more expressive